specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/capacityReservation.json

```diff
@@ -0,0 +1,1015 @@
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "capacityReservationGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the capacity reservation group."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CapacityReservationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List capacity reservations in reservation group.": {
+            "$ref": "./examples/capacityReservationExamples/CapacityReservation_ListByReservationGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CapacityReservationGroupInstanceView": {
+      "type": "object",
+      "properties": {
+        "capacityReservations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CapacityReservationInstanceViewWithName"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "readOnly": true,
+          "description": "List of instance view of the capacity reservations under the capacity reservation group."
+        },
+        "sharedSubscriptionIds": {
+          "type": "array",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResourceReadOnly"
+          },
+          "x-ms-identifiers": [],
+          "readOnly": true,
+          "description": "List of the subscriptions that the capacity reservation group is shared with. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details."
+        }
+      }
+    },
+    "ResourceSharingProfile": {
+      "type": "object",
+      "properties": {
+        "subscriptionIds": {
+          "type": "array",
+          "description": "Specifies an array of Subscription resource Ids that capacity reservation group is shared with. The resource Id should be provided in form: /subscriptions/{subscriptionId}. **Note:** Minimum api-version: 2023-09-01. Please refer to https://aka.ms/computereservationsharing for more details.",
+          "items": {
+            "$ref": "../../../common-types/v1/common.json#/definitions/SubResource"
+          }
```

Comment:
Would calling them 'subscriptionResourceIds' be helpful to developers for understanding they need to pass the full resource id "/subscriptions/guid" instead of just the guid part?
Or any other name besides just 'subscriptionIds'

Reply:
i agree with the confusing part. i just removed 
"The resource Id should be provided in form: /subscriptions/{subscriptionId}" all together

Reply:
for visibility @nreact

Reply:
updated in the latest comment as advised